src/keria/app/aiding.py

```diff
@@ -1914,11 +1914,15 @@ def on_get(req, rep, name):
 
         signing = []
         for smid in smids:
+            if isinstance(smid,dict):
+                smid = smid['i']
```

Comment:
Are there really cases where the contents of smids could be either a dict or a str?  We should figure out why and if that's necessary